docs/api_docs/bundle.yaml

```diff
@@ -927,6 +927,21 @@ paths:
           description: generic error response
           schema:
             $ref: '#/definitions/error'
+  /migrations:
+    get:
+      tags:
+        - migration
+      operationId: runMigrations
```

Comment:
nit, if it's GET, should this be something like getMigrations?

Reply:
yeah, I was a little unsure on this, i'm using this as endpoint to run the migration process. It will only run the migrations if finds new files in the directory